### ISSUE TITLE
 After finetune, there is an action gap during inference on my own dataset.

### ISSUE THREAD

**Issue by HanwenWangBUPT** [openvla/openvla#126]
Hi,
Thank you for your outstanding work.
As the title suggests, my fine-tuning results are poor. When inferring using the training dataset, there is a significant difference between the results and the action labels. I did not modify the fine-tuning code, and during inference, I adhered to the input format for images and used the same resizing method as the official one. I also mapped the action outputs to the range of actions in my dataset using q01 and q99 for un-normalization. During fine-tuning, I did not use image augmentation, and my action accuracy reached 1. I am quite puzzled about where the problem lies. 

here an example ,
infer result: [1.23738531e-03  2.06876544e-03  4.49192716e-03 -4.82013702e-04 -8.72295174e-03  1.81586735e-03  9.98400000e+02]
label: [7.24270521e-03 -2.33919453e-03 -1.92745328e-02 -7.46141002e-03 4.27616847e-04  1.58309005e-02  3.42000000e+02]

**Comment by moojink** [comment]
Hi @HanwenWangBUPT,

That is indeed strange. If you observe 100% train accuracy and run model inference on training samples, you should observe perfect predictions that match the labels.

It's hard for us to help debug without more information, but as a debugging step, it may help to try overfitting to a single sample (literally one datapoint) and seeing if you can even predict the action tokens correctly for this one sample. Then try it again with five samples. After you pass these tests, you can proceed with your original dataset.

- Moo Jin
